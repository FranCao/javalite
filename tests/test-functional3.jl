void print1()
{
    print("hello");
}

print(1);

void print2()
{
    print("hi");
}

print(2);

void doNothing() {}

doNothing();
print1();
print2();

