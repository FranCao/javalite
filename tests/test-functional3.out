1
2
hello
hi